build.sbt

```diff
@@ -24,6 +24,7 @@ inThisBuild(
 )
 
 addCommandAlias("build", "; prepare; testJVM")
+addCommandAlias("prepare", "fmt")
```

Comment:
Should we just have `fmt` so there is only one way to do the same thing?

Reply:
cc @sideeffffect :). As I mentioned before, I'm fine with either as long as all aliases are working.

Reply:
Personally, I got used to just using `fix` to "fix" all the issues that can be automatically fixed, regardless of whether we use Scalafix, Scalafmt, or both.
Thus I've never used `prepare`.
But I don't have a strong opinion on this.

Reply:
Ok, in that case I'm removing the `prepare` as well, and I'll replace it directly in `build`.